src/rgw/rgw_http_client.cc

```diff
@@ -253,7 +382,8 @@ int RGWHTTPClient::process(const char *method, const char *url)
   last_method = (method ? method : "");
   last_url = (url ? url : "");
 
-  curl_handle = curl_easy_init();
+  auto ca = handles->get_curl_handle();
```

Comment:
Since the use case is almost exclusively for keystone, do we have added benefit using the curl-share interface? https://curl.haxx.se/libcurl/c/libcurl-share.html